src/dash-table/dash/DataTable.js

```diff
@@ -595,52 +579,46 @@ export const propTypes = {
     }),
 
     /**
-     * DEPRECATED
-     */
-    navigation: PropTypes.string,
-
-    /**
-     * `column_conditional_dropdowns` specifies the available options
-     * for dropdowns in various columns and cells.
-     * This property allows you to specify different dropdowns
-     * depending on certain conditions. For example, you may
-     * render different "city" dropdowns in a row depending on the
-     * current value in the "state" column.
+     * `dropdown` represents the available dropdown
+     * options for different columns.
+     * The `id` property refers to the column ID.
+     * The `dropdown` property refers to the `options` of the
+     * dropdown.
      *
      * NOTE: The naming and the behavior of this option may change
      * in the future.
      * Tune in to [https://github.com/plotly/dash-table/issues/168](https://github.com/plotly/dash-table/issues/168)
      */
-    column_conditional_dropdowns: PropTypes.arrayOf(PropTypes.shape({
-        id: PropTypes.string.isRequired,
-        // .exact
-        dropdowns: PropTypes.arrayOf(PropTypes.shape({
-            condition: PropTypes.string.isRequired,
-            // .exact
-            dropdown: PropTypes.arrayOf(PropTypes.shape({
-                label: PropTypes.string.isRequired,
-                value: PropTypes.oneOfType([
-                    PropTypes.number,
-                    PropTypes.string
-                ]).isRequired
-            })).isRequired
+    dropdown: PropTypes.objectOf(PropTypes.exact({
+        clearable: PropTypes.bool,
+        dropdown: PropTypes.arrayOf(PropTypes.exact({
```

Comment:
should we call this `options`? (and all the other arrays of `{label, value}` like it in other `*dropdown*` props) That would match `dcc.Dropdown`, also `dropdown.{colID}.dropdown` seems weird.

Reply:
It does feel weird. Since I had 2 `dropdown` and 1 `options` I went from `dropdown`! Updating.